backend/graminsta/post/views.py

```diff
@@ -65,4 +66,4 @@ def get(request, username):
             Users that follows the given user
         """
         following_people = get_people_user_follows(username)
-        return Response(following_people.data)
+        return Response(UserSerializer(following_people, many=True).data)
```

Comment:
Not related to this PR, but keep in mind that we might need to revise the information we returned to client, i.e. are we returning too much information that might have a risk to break user privacy (age, gender, etc.).